Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -704,6 +705,8 @@ def identitySim(wf):
 # INPUT command for reminiAOD wfs on UL-like relval inputs
 steps['TTbar_13_reminiaod2016UL_preVFP_INPUT']={'INPUT':InputInfo(dataSet='/RelValTTbar_13UP16/CMSSW_10_6_12-PU25ns_106X_mcRun2_asymptotic_preVFP_v8_hltul16_preVFP-v1/AODSIM',label='rmaod',location='STD')}
 steps['TTbar_13_reminiaod2016UL_postVFP_INPUT']={'INPUT':InputInfo(dataSet='/RelValTTbar_13UP16/CMSSW_10_6_12-PU25ns_106X_mcRun2_asymptotic_v13_hltul16_postVFP-v1/AODSIM',label='rmaod',location='STD')}
+# INPUT command for reminiAOD wfs on PbPb relval inputs 
+steps['HydjetQ_reminiaodPbPb2018_INPUT']={'INPUT':InputInfo(dataSet='/RelValHydjetQ_B12_5020GeV_2018_ppReco/CMSSW_10_3_3-PU_103X_upgrade2018_realistic_HI_v11-v1/GEN-SIM-RECO',label='rmaod',location='FNAL')}
```

Comment:
I think that the location should still be STD,
even though it apparently doesn't matter
https://github.com/cms-sw/cmssw/blob/9a46186eea1ef61c1e2f1b690c5c38eee2976ac2/Configuration/PyReleaseValidation/python/MatrixUtil.py
will still point to T2_CH_CERN
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-e6456f/7332/runTheMatrix-results/158.11_HydjetQ_reminiaodPbPb2018_INPUT+HydjetQ_reminiaodPbPb2018_INPUT+REMINIAODHI2018PPRECO+HARVESTHI2018PPRECOMINIAOD/cmdLog